summingbird-storm/src/main/scala/com/twitter/summingbird/storm/collector/AggregatorOutputCollector.scala

```diff
@@ -93,24 +96,23 @@ class AggregatorOutputCollector[K, V: Semigroup](
     }
   }
 
-  private def trackMessageId(tuple: (K, V), o: scala.Any, s: String): Unit = {
+  private def trackMessageId(tuple: (K, V), o: AnyRef, s: String): Unit = {
     val messageIdTracker = streamMessageIdTracker.getOrElse(s, MMap[Int, MList[Object]]())
```

Comment:
`.getOrElseUpdate` here and you won't have to put it back in at the end on line 103 (which will save work).